client/src/components/AddMates/AddMateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  DeleteOutlined,
+  EllipsisOutlined,
+  UserOutlined,
+} from "@ant-design/icons";
+import { Avatar, Card } from "antd";
+import React, { FC } from "react";
+
+import { AddMateType } from "../types";
+
+type AddMateCardProps = {
```

Comment:
kindof sounds like a verb rather than a noun, maybe rename to something like newMateCard

Reply:
This was to follow convention of naming the proptypes of `ComponentName` to `ComponentNameType`, but yes kinda confusing, renamed.